scheduler/src/cook/mesos.clj

```diff
@@ -214,10 +217,12 @@
                                         (sched/create-datomic-scheduler
                                           mesos-datomic-conn
                                           (fn set-or-create-framework-id [framework-id]
-                                            (curator/set-or-create
-                                              curator-framework
-                                              zk-framework-id
-                                              (.getBytes (-> framework-id mesomatic.types/pb->data :value) "UTF-8")))
+                                            (let [value (-> framework-id mesomatic.types/pb->data :value)]
```

Comment:
I added an error log if it's being set to a different value than what `framework-id-promise` already has